apps/openmw_test_suite/detournavigator/recastmeshbuilder.cpp

```diff
@@ -310,11 +310,19 @@ namespace
             AreaType_ground
         );
         const auto recastMesh = builder.create();
+#ifdef BT_USE_DOUBLE_PRECISION
+        EXPECT_EQ(recastMesh->getVertices(), std::vector<float>({
```

Comment:
Maybe it is worth to use the `EXPECT_NEAR` with something like the 1e-06 epsilon instead. In this case we do not need to use additional CMake flags at all.

Reply:
`EXPECT_NEAR ` doesn't support containers, but `EXPECT_THAT` with `FloatNear` can do this. CMake flags and defines are still required because bullet headers don't know about used precision.

Reply:
> CMake flags and defines are still required because bullet headers don't know about used precision.

But you do not use these flags anyway now, and there is still no agreement about how to configure and handle `BT_USE_DOUBLE_PRECISION`.
I suggest to remove the `BT_USE_DOUBLE_PRECISION` from this PR and merge it to 0.46, and then continue discussion in the separate PR's thread, when we really will need this flag.

Reply:
Done